apprise/plugins/NotifyPushover.py

```diff
@@ -194,12 +194,23 @@ class NotifyPushover(NotifyBase):
             'regex': (r'[a-z]{1,12}', 'i'),
             'default': PushoverSound.PUSHOVER,
         },
+        'retry': {
+            'name': _('Retry'),
+            'type': 'int',
+            'min': 30,
+        },
+        'expire': {
+            'name': _('Expire'),
+            'type': 'int',
+            'max': 10800,
```

Comment:
Since you later check for a min of zero (0), can you just add another entry here: `'min': 0,`

Reply:
in def url I append retry and expire only if emergency
```
        if self.priority == PushoverPriority.EMERGENCY:
            args.update({'expire': self.expire, 'retry': self.retry})
```

I will add the min 0, but I was unable to find out how to use these min and max's to verify the parameters. Which is why I added the manual check, as in 
```
expire < 0 or expire > 10800
```